static/js/hotkey.js

```diff
@@ -214,6 +214,10 @@ export function in_content_editable_widget(e) {
 
 // Returns true if we handled it, false if the browser should.
 export function process_escape_key(e) {
+    if (hashchange.in_recent_topics_hash() && recent_topics.change_focused_element(e, "escape")) {
+        return true;
+    }
```

Comment:
Why does this logic need to be moved?

Reply:
To keep all the escape key handing in one place.